### PR TITLE
Feature/contact upcoming sessions

### PR DESCRIPTION
**Pull request by timohuber**
Include experiments with upcoming assignments in contact experiment history

### CODE REVIEW

File: .devcontainer/devcontainer.json
Comment (mabiede) [inline .devcontainer/devcontainer.json:1]: Is this file intended to be changed?

-> Empty newline at EOF

File: resources/index.scss [hunk 1]
```diff
@@ -303,6 +303,35 @@ button.as-link {
     }
 }
 
+// Remove once framework is released
```
Comment (mabiede) [inline resources/index.scss:306]: is there a corresponding issue in the framework project?

Comment (timohuber) [reply resources/index.scss:306]: Yes it is merged and will be released soon

**Review by mabiede — APPROVED** [review]
